dialoptions.go

```diff
@@ -55,13 +55,15 @@ type dialOptions struct {
 	// balancer, and also by WithBalancerName dial option.
 	balancerBuilder balancer.Builder
 	// This is to support grpclb.
-	resolverBuilder      resolver.Builder
-	reqHandshake         envconfig.RequireHandshakeSetting
-	channelzParentID     int64
-	disableServiceConfig bool
-	disableRetry         bool
-	disableHealthCheck   bool
-	healthCheckFunc      internal.HealthChecker
+	resolverBuilder         resolver.Builder
+	reqHandshake            envconfig.RequireHandshakeSetting
+	channelzParentID        int64
+	disableServiceConfig    bool
+	disableRetry            bool
+	disableHealthCheck      bool
+	healthCheckFunc         internal.HealthChecker
+	defaultServiceConfig    *ServiceConfig
+	defaultServiceConfigRaw string
```

Comment:
Add a field to `ServiceConfig` for the raw js string.

Reply:
done